x-pack/plugin/src/yamlRestCompatTest/resources/rest-api-spec/api/xpack-ssl.certificates.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Is this meant to be identical to https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/ssl.certificates.json?

Reply:
Ah, these routes aren't meant to be generated by clients, gotcha! Disregard my comment then.